### DISCUSSION TITLE
How do I change the favicon?

### DISCUSSION THREAD

**Discussion by BogusCurry** [datenstrom/community#750] [Ask a question] [answered]
Hello ;D
Short question, how I can change the favoriteicon ? I have the actual version from yellow ;D 

Sorry for my englisch, I am comes from germany ;D

Greetings

Bogus | htttps://www.magicplace.de

**Comment by schulle4u** [comment, marked as answer]
Yellow automatically looks for the file `$theme.png` in your `system/themes` directory, meaning it should have the same name as your installed theme, e. g. `berlin.png`. 

See also: [How to customise a theme](https://datenstrom.se/yellow/help/how-to-customise-a-theme)

**Comment by BogusCurry** [comment]
Ahh ok, thanks for your fast answer ;D

**Reply by schulle4u** [reply]
You're welcome. And please mark an answer if your question has been answered. 🙂
